src/ripple/app/main/Application.cpp

```diff
@@ -1599,7 +1599,7 @@ ApplicationImp::loadLedgerFromFile (
         {
             Json::Value& entry = ledger.get()[index];
 
-            if (!entry.isObject())
+            if (!entry.isObjectorNull())
```

Comment:
Suggestion: capitalize `or` eg. `isObjectOrNull`

Upon reading, I immediately thought what is an `Objector`?